### PR TITLE
Avoid `None` as a key in the response JSON.

### PR DESCRIPTION
**Pull request by ShifaSZ**
The `create_json_response` function calls the `JsonResponse` function with `sort_keys` as the JSON dump params. In this case, `None` as a key in the JSON is not allowed. Converting `None` to a string `'null'` aligns the `json.dumps` behavior without the `sort_keys` param.

### CODE REVIEW

**Comment by hanars** [comment]
How does this actually show to the user? My gut would be that this will look wrong and we are better off filtering these entirely, but depending on how they look it might be okay?

**Comment by ShifaSZ** [comment]
I got a screenshot for the UI of a `None` gene_id transcript in the popup transcript modal, as shown below.
image

**Comment by hanars** [comment]
What does the variant itself look like, these are SVs so they should be showing all the genes without clicking into the modal

But "Gene Id: null" is certainly not acceptable to show the user

**Comment by ShifaSZ** [comment]
It is an SV, and it has many transcripts/genes.
image

**Comment by hanars** [comment]
so what does in look like when you expand the genes?

**Comment by ShifaSZ** [comment]
It shows thousands of genes. The screenshot below shows the last few of them. You can see the transcript data on the right side of the screenshot.
image

**Comment by hanars** [comment]
I really need to be able to see how these new transcripts will be shown to the user. You either need to find an example with fewer genes, provide a screenshot that actually shows the "null" genes, or schedule time with me to meet. Showing just the end of the screenshot is not sufficient for me to figure out what changes are necessary

**Comment by ShifaSZ** [comment]
In the VariantGenes component, there is a statement as the following. Since genesById['null'] is undefined, the transcripts with the `null` gene id will be filtered out.
https://github.com/broadinstitute/seqr/blob/c200fdc3eee153efde298027946099be92cde59c/ui/shared/components/panel/variants/VariantGene.jsx#L699

**Comment by ShifaSZ** [comment]
The issue with the first option is that we don't have gene details.  We can only display the information in the `transcripts` field of the variant. So I also prefer the second option.

File: seqr/utils/search/elasticsearch/es_utils_tests.py [hunk 1]
```diff
@@ -138,7 +138,12 @@ def call_request_json(self, index=-1):
               'consequence_terms': [
                   'frameshift_variant'
               ]
-            }
+            },
+            {
```
Comment (hanars) [inline seqr/utils/search/elasticsearch/es_utils_tests.py:142]: it would be better/ more accurate to add this to the mock SV variant, instead of here

**Review by hanars — APPROVED** [review]

**Pull request merged**